### ISSUE TITLE
"Go to File" prompt is announced repeatedly for each typed character when using screen readers (Orca, NVDA, Narrator)

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#279411]
Type: <b>Bug</b>

When using a screen reader (Orca on Linux, NVDA on Windows, or Narrator on Windows) and pressing Ctrl+P to open the Go to File command, the initial prompt is announced correctly.
However, as you type each character to search for a file, the screen reader repeats the prompt every time a character is entered. This creates unnecessary verbosity and significantly impacts productivity.
 
Steps to Reproduce
1. Open VS Code.
2. Enable a screen reader (Orca, NVDA, or Narrator).
3. Press Ctrl+P to open the Go to File command.
4. Start typing a file name.
5. Notice that the prompt is announced again for every character typed.
 
Actual Behavior
The screen reader announces the prompt repeatedly for each character typed.
 
Expected Behavior
The screen reader should announce the prompt only once when the command is activated. After that, it should announce only relevant changes, such as search results or selection updates, not repeat the prompt.

VS Code version: Code - Insiders 1.107.0-insider (51e7119ccb43f1e1dbb3d059bfc9bac09eebb8cb, 2025-11-25T07:07:56.708Z)
OS version: Windows_NT x64 10.0.26200
Modes:
Remote OS version: Linux x64 6.6.87.2-microsoft-standard-WSL2
Remote OS version: Linux x64 6.6.87.2-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: disabled_software<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: disabled_software<br>video_encode: disabled_software<br>webgl: disabled_off<br>webgl2: disabled_off<br>webgpu: disabled_off<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.66GB (3.66GB free)|
|Process Argv|--crash-reporter-id f8460393-7b6d-4fa3-b912-843cc3297a42|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: FedoraLinux-42|
|OS|Linux x64 6.6.87.2-microsoft-standard-WSL2|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 0)|
|Memory (System)|7.59GB (5.36GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: FedoraLinux-42|
|OS|Linux x64 6.6.87.2-microsoft-standard-WSL2|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 0)|
|Memory (System)|7.59GB (5.37GB free)|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-containers|ms-|0.433.0
remote-ssh|ms-|0.122.2025111815
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.104.3
vscode-remote-extensionpack|ms-|0.26.0
remote-explorer|ms-|0.6.2025081809
remote-server|ms-|1.6.2025102109
copilot-chat|Git|0.34.2025112501
bash-ide-vscode|mad|1.43.0
shfmt|mkh|1.5.1
java|red|1.49.0
vscode-gradle|vsc|3.17.1
vscode-java-debug|vsc|0.58.3
vscode-java-dependency|vsc|0.26.5
vscode-java-test|vsc|0.43.2
vscode-maven|vsc|0.45.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
pythonvspyt551cf:31249598
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1cf:31399617
6518g693:31302842
82j33506:31327384
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
aihoversummaries_t:31413033
todos-1:31405333
qwen_all_req:31396049
3efgi100_wstrepl:31403338
trigger-command-fix:31379601
auto_model_enabled:31385282
use-responses-api:31390341
sonnet_45_3e700918:31424416
je187915:31401257
nes-next-cursor-off:31426496
3i15c187:31419923
disable_auto_fix_gb0j8799:31411651
agentic_ask_h7d75428:31406124
copilot-nes-oct-t:31418112
terminalsuggestenabled:31421703
none:31426257
cp_cls_c_966_ss:31426491
chatvisible:31417128
expandedtokens:31418009
inlinechat_v2_hd992725:31418437
decaj161:31422462
find_all_ref_in_bg_t:31420845
3fia6346:31423209
478ah919:31426269

```

</details>

**Comment by TylerLeonhardt** [comment]
@jvesouza does this feel like a regression for you? Did this use to not work this way?
